.github/workflows/python.yml

```diff
@@ -8,6 +11,7 @@ on:
 
 jobs:
   check-missing-migration:
+    if: ${{ github.ref != 'refs/heads/master' }}
```

Comment:
If you do the above, I guess this needs to be updated too.

Reply:
It's fine, not running this on master/main is the low hanging fruit.

Reply:
I really don't want this thing to be chiming in on release branches when I'm cutting a release in the future. We can do this later but I feel like it is a must, not a nice to have.

Reply:
nm, will just do it in this PR. https://github.com/getsentry/sentry/pull/20537#discussion_r498442637